ui/yarn.lock

```diff
@@ -3063,7 +3063,8 @@ anymatch@~3.1.2:
     picomatch "^2.0.4"
 
 "api-common-protos@link:./lib/api-common-protos":
-  version "1.0.0"
+  version "0.0.0"
+  uid ""
```

Comment:
**[observation, non-blocking]**

My version of Yarn does this too and I’ve never been able to figure out why. I don’t _think_ it’s a problem, just a little mysterious.